samples/nft_deployment/nodes/alice/core.yaml

```diff
@@ -0,0 +1,295 @@
+  # Keepalive settings for node server and clients
+  keepalive:
+    # MinInterval is the minimum permitted time between client pings.
+    # If clients send pings more frequently, the peer server will
+    # disconnect them
+    minInterval: 60s
+    # Interval is the duration after which if the server does not see
+    # any activity from the client it pings the client to see if it's alive
+    interval: 300s
+    # Timeout is the duration the server waits for a response
+    # from the client after sending a ping before closing the connection
+    timeout: 600s
+  # P2P configuration
+  p2p:
+    # Listening address
+    listenAddress: /ip4/127.0.0.1/tcp/20007
+    # If empty, this is a P2P boostrap node. Otherwise, it contains the name of the FCS node that is a bootstrap node
+#    bootstrapNode: issuer
+  # The Key-Value Store is used to store various information related to the FSC node
+  kvs:
+    persistence:
+      # Persistence type can be \'badger\' (on disk) or \'memory\'
+      type: badger
+      opts:
+        path: /Users/bur/Developer/gocode/src/github.com/hyperledger-labs/fabric-token-sdk/samples/nft/testdata/fsc/nodes/alice/kvs
+    cache:
+        # Sets the maximum number of cached items 
+        size: 200
+
+  web:
+    enabled: true
+    # HTTPS server listener address
+    address: 0.0.0.0:20008
+    tls:
+      enabled: false
+  tracing:
+    provider: none
+    udp:
+      address: 127.0.0.1:8125
+  metrics:
+    # metrics provider is one of statsd, prometheus, or disabled
+    provider: none
+    # statsd configuration
+    statsd:
+      # network type: tcp or udp
+      network: udp
+      # statsd server address
+      address: 127.0.0.1:8125
+      # the interval at which locally cached counters and gauges are pushed
+      # to statsd; timings are pushed immediately
+      writeInterval: 10s
+      # prefix is prepended to all emitted statsd metrics
+      prefix:
+
+  # The endpoint section tells how to reach other FSC node in the network.
+  # For each node, the name, the domain, the identity of the node, and its addresses must be specified.
+  endpoint:
+    resolvers: 
+    - name: issuer
+      domain: fsc.example.com
+      identity:
+        id: issuer
+        path: /Users/bur/Developer/gocode/src/github.com/hyperledger-labs/fabric-token-sdk/samples/nft/testdata/fsc/crypto/peerOrganizations/fsc.example.com/peers/issuer.fsc.example.com/msp/signcerts/issuer.fsc.example.com-cert.pem
+      addresses: 
+         P2P: 127.0.0.1:20001
+      aliases: 
+    - name: auditor
+      domain: fsc.example.com
+      identity:
+        id: auditor
+        path: /Users/bur/Developer/gocode/src/github.com/hyperledger-labs/fabric-token-sdk/samples/nft/testdata/fsc/crypto/peerOrganizations/fsc.example.com/peers/auditor.fsc.example.com/msp/signcerts/auditor.fsc.example.com-cert.pem
+      addresses: 
+      aliases: 
+    - name: bob
```

Comment:
a little confuse, why bob is here in alice's config file?

Reply:
The `fsc.endpoint.resolvers` section of the config is used to authenticate the FSC nodes when interacting. Even though the node network component comes with a discovery mechanism, extra authentication is needed that only known parties join the FSC network